Regression/WarpX-tests.ini

```diff
@@ -1515,7 +1515,7 @@ tolerance = 1.e-14
 [Maxwell_Hybrid_QED_solver]
 buildDir = .
 inputFile = Examples/Tests/Maxwell_Hybrid_QED/inputs_2d
-runtime_params = warpx.cfl = 0.7071067811865475
```

Comment:
I think that it would make sense to put `algo.maxwell_solver=psatd` directly in the input script `Maxwell_Hybrid_QED/inputs_2d`, since the Maxwell QED module can only be used with PSATD anyway right now.

Reply:
Sounds good, yes.

Do you know why we set the `warpx.cfl=0.7071067811865475` for this test only in runtime options? Could move that to the inputs script, too.

Reply:
Hm, for this particular test, I don't think that `warpx.cfl=0.7071067811865475` is important : my guess is that it was copy from some other test where this CFL was important. (I can give more details offline.)

However, we need to keep this value of the CFL for this test for now, unless we want to also update the corresponding reference checksum (which is somewhat orthogonal to this PR).
I agree that, for this particular test, we could move the `warpx.cfl` to the input script.

Reply:
Ok, seems like we could update this generally and I'll leave it for this PR where it stands.